src/routes/quickstarts/react-native/react-native-testing-library.mdx

```diff
@@ -0,0 +1,242 @@
+# Mock Network Requests in React Native Testing Library with Mirage
+
+Use your Mirage server to test your React Native application under different server scenarios using [React Native Testing Library](https://github.com/testing-library/native-testing-library) and [Jest Native](https://github.com/testing-library/jest-native).
+
+> This is a quickstart guide should also work for people using [`react-test-renderer`](https://reactjs.org/docs/test-renderer.html) or [`react-native-testing-library`](https://github.com/callstack/react-native-testing-library).
+
+## Step 1: Install Mirage
+
+First, make sure you have Mirage installed:
+
+```bash
+# Using npm
+npm install --save-dev miragejs
+
+# Using Yarn
+yarn add --dev miragejs
+```
+
+## Step 2: Setup your testing environment
+
+Out of the box, React Native does a good job to polyfill browser API's that we might need in libraries that were originally built to run on the browser. Our problem when using Mirage in a Jest testing environment is that we don't have access to some properties that the browser provides by default. To get around this issue we must update Jest's environment immediately after its setup is finished.
+
+To update your Jest environment follow these steps:
+
+1. Create a setup file in the root of your project. For the purpose of this guide we'll call it `jest.setup.js`.
+
+2. Install a polyfill for XMLHttpRequest:
+
+```bash
+# Using npm
+npm install --save-dev xmlhttprequest
+
+# Using Yarn
+yarn add --dev xmlhttprequest
+```
+
+3. Patch the variable `global` in your `jest.setup.js` to add support for XMLHttpRequest, self and window.
+
+```js
+// jest.setup.js
+global.self = global;
+global.window = {};
+global.XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest;
+```
+
+4. Tell Jest to load the environment that we've created by adding the [`setupFilesAfterEnv`](https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array) to your Jest configuration. We must add `jest.setup.js` as an item in the list of files that Jest must execute after setting up its environment.
```

Comment:
I think you can move "We must add `jest.setup.js` as an item in the list of files that Jest must execute after setting up its environment." to the bottom of the code block and say something along the lines of...

"Here we're telling Jest to execute our `jest.setup.js` after it sets up the test environment."

Reply:
👍